### DISCUSSION TITLE
Any ideas why we could see this warning message?

### DISCUSSION THREAD

**Discussion by jzs11** [vuejs/language-tools#1259] [General]
![image](https://user-images.githubusercontent.com/2770401/166810950-7beaf8fd-bad2-4c08-96e0-609baed2a880.png)

**Comment by yaegassy** [comment]
Check this issue and commit comments.

- <https://github.com/johnsoncodehk/volar/issues/1124>
- <https://github.com/johnsoncodehk/volar/commit/afd4f4ba3a34b939f158b63ac72729aec65f9cb3#commitcomment-72663442.>

**Comment by jzs11** [comment]
![image](https://user-images.githubusercontent.com/2770401/167101250-453fa385-82b0-4d49-8602-7111fdb6d1ad.png)
🎉

**Comment by jzs11** [comment]
@johnsoncodehk those warnings come back again.

![image](https://user-images.githubusercontent.com/2770401/170803755-934e46e6-62aa-45b9-a52d-d6004e318a05.png)

**Reply by johnsoncodehk** [reply]
Same to https://github.com/johnsoncodehk/volar/issues/1124#issuecomment-1123312339.

**Reply by jzs11** [reply]
That's really helpful, thanks!
